corehq/apps/toggle_ui/views.py

```diff
@@ -200,7 +200,10 @@ def _get_usage_info(toggle):
                 active_domains += 1
         else:
             try:
-                last_used[name] = _format_date(CouchUser.get_by_username(name).last_login)
+                user = CouchUser.get_by_username(name)
+                if not user:
+                    raise ResourceNotFound
+                last_used[name] = _format_date(user.last_login)
```

Comment:
nitpicky - I see you're trying to avoid duplicating the `NOT_FOUND` assignment logic, but raising the arbitrary error that we know is going to get caught also feels a little messy.

I might favor just writing this as:

```python
last_used[name] = _format_date(user.last_login) is user else NOT_FOUND
```